pyspecProcScripts/phasing.py

```diff
@@ -302,7 +302,6 @@ def hermitian_function_test(
     # I tried removing the following line, and the 2D residual actually
     # looks sharper, but then the 1D plot of the cost function disappears
     # -- what's up with that?
-    residual /= abs(center_point) # weight 'residual' by 1/(signal amplitude)
```

Comment:
so... I'm not positive if this is a good thing for not (probably best to get @aabeaton to test on some real data), but removing this does give a much sharper cost function